src/sentry/testutils/factories.py

```diff
@@ -748,7 +749,9 @@ def _sentry_app_kwargs(**kwargs):
         return _kwargs
 
     @staticmethod
-    def create_sentry_app_installation(organization=None, slug=None, user=None, status=None):
+    def create_sentry_app_installation(
+        organization=None, slug=None, user=None, status=None, prevent_token_exchange=False
+    ):
```

Comment:
`prevent_token_exchange` -> when we want to test the GrantExchange mediator independantly

Reply:
@NisanthanNanthakumar how does this get set to `True`? I see it in tests but how is it going to be used by production code?

Reply:
@NisanthanNanthakumar oh duh this is in testutils, my bad